usr/src/uts/common/fs/zfs/dmu_objset.c

```diff
@@ -67,6 +67,13 @@ krwlock_t os_lock;
  */
 int dmu_find_threads = 0;
 
+/*
+ * Backfill lower metadnode objects after this many have been freed.
+ * Backfilling negatively impacts object creation rates, so only do it
+ * if there are enough holes to fill.
+ */
+int dmu_rescan_dnode_threshold = 1 << DN_MAX_INDBLKSHIFT;
```

Comment:
This variable is a count of dnodes, right?  I don't understand what that has to do with the maximum indirect block size (in bytes).